client/components/payment-method-details/index.js

```diff
@@ -6,6 +6,8 @@
 import { Fragment } from 'react';
 import './style.scss';
 import p24BankList from '../../payment-details/payment-method/p24/bank-list';
+import { HoverTooltip } from '../tooltip';
+import { PAYMENT_METHOD_TITLES } from 'payment-methods/constants';
```

Comment:
We already have `PaymentMethodInformationObject` which contains translated title, accessible by payment method id. Can it be used here instead of a new constant?

Reply:
On a separate thought: can brands be a part of the `PaymentMethodInformationObject` records? It would allow to have all the payment method properties in one place. And, it would clearly indicate which brand belongs to which payment method, for devs at least.

Reply:
Thanks for the review and feedback, @dmvrtx 🙌 

> Anyways, can we at least use translations inside the PAYMENT_METHOD_TITLES constant?

I was a bit divided about adding translations for brand names, since for eg. `Affirm` will be known as same in all languages, since it is a brand. Is it still recommended we add translation tags?

Reply:
>On a separate thought: can brands be a part of the PaymentMethodInformationObject records? It would allow to have all the payment method properties in one place. And, it would clearly indicate which brand belongs to which payment method, for devs at least.

The `PaymentMethodInformationObject` contains a list of PM configs. While most of them have a single brand associated with them, a few have multiple brands in them.. specifically so - Cards - ( Visa, Master, JCB, Amex, Discover etc ) . To help clarity for devs, do we want to add one more array property of `brands` or say `brandTitles` with all possible brands for that PM within it  ( or a single value for those that do not have multiple brands ) ?

Reply:
I like the `brandTitles` approach. It also made me think if we can somehow use it for "Afterpay/Clearpay" branding.

Reply:
Added translation tags here - 748cfb7

Added brandTitles property to the `PaymentMethodInformationObject`  here 49d11cc

Reply:
I may miss something, but it seems that the `brandTitles` are not being used?

Reply:
Hi Miguel. The `brandTitles` is useful to show a developer exploring `PaymentMethodInformationObject`, what are the different brand names that could be associated with the Payment method. Eg. there are many brand names that are associated with the `card` method. 

The current PR merely adds it since it came up in a discussion. ( probably not the best way to make additions to code, but this one was a small change ) .. 

The PR use case here requires the brand id to be changed to a proper humanly readable brand name with correct capitalization.. Eg. `visa` should get converted to `Visa` , `affirm` to `Affirm` .  Hence, `brandTitles` will probably help future devs find out what brands are associated with the PM, however it does not help us here to resolve a brand ID to a brand name. I hope this is making sense.